### PR TITLE
[T4A3][T11-B1]Paul Tan Han Kiat

### PR DESCRIPTION
**Pull request by pyokagan**
Ready for review.

### CODE REVIEW

**Comment by xdshang** [comment]
Accepting this PR without a review because another of your PRs for this tutorial was reviewed in detail.

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    public final String value;
+    private final boolean isPrivate;
+
+    public Contact(String value, boolean isPrivate) {
+        this.value = value.trim();
+        this.isPrivate = isPrivate;
+    }
+
+    @Override
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Contact.java:16]: What is the meaning to have `@Override` in the base class?

**Comment by xdshang** [comment]
@pyokagan Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Contact.java (hunk 1 above)
Comment (pyokagan) [reply src/seedu/addressbook/data/person/Contact.java:16]: @xdshang To ensure that the method is overriding something, in this case the `toString`, `equals` and `hashCode` methods of the base `Object` class. AFAIK, all Java classes inherit from `Object`.

**Pull request closed without merging**